collator/src/lib.rs

```diff
@@ -162,6 +162,31 @@ where
 			.ok()
 	}
 
+	/// Returns channels contents for each inbound HRMP channel addressed to the parachain we are
+	/// collating for.
+	///
+	/// Empty channels are also included.
+	fn retrieve_inbound_hrmp_channels_contents(&self, relay_parent: PHash)
```

Comment:
nit: colloquially it reads more 'correctly' to say `hrmp_channel_contents`, even though it is reading many channels. You might change it to `all_hrmp_channel_contents` or similar if conveying the "all" aspect is important.

Reply:
oh thanks! I did want to emphasize that it's multiple channels but I also didn't like how `hrmp_channels_contents`. `all_hrmp_channel_contents` indeed sounds better. Changed!